4.analyze_data/notebooks/linear_model/0.fit_linear_model_strat.ipynb

```diff
@@ -19,7 +19,7 @@
     "from sklearn.linear_model import LinearRegression\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from sklearn.linear_model import LinearRegression
</pre><p>Consider removing <code>LinearRegression</code> if you are no longer using it </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/CFReT_data/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
This is used within the notebook in the last cell.